test/TEST-13-NSPAWN-SMOKE/create-busybox-container

```diff
@@ -5,8 +5,11 @@ set -u
 set -o pipefail
 
 root="${1:?Usage $0 container-root}"
-mkdir -p "$root"
-mkdir "$root/bin"
+if [[ -d "$root" ]]; then
+    echo "$root already exists"
+    exit 0
```

Comment:
> Also sligthly simplify things by not creating separate chroot for each test
run, the same one can be reused over and over

Well, the behaviour of the `-U`-option depends on the `$root`'s uid and gid. I think we should create a separate root to really test `-U`